### DISCUSSION TITLE
Is there anyway to mount and modify the `stateful_partition` without a shim?

### DISCUSSION THREAD

**Discussion by s0urce-c0de** [3kh0/ext-remover#1142] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Read the title &uarr;&uarr;&uarr;&uarr;

**Comment by nathblade16** [comment]
I'm sorry but I have not really touched chromeOS for quite a while now and focus on linux and repurposing chromeOS devices as linux laptops, can you please give me a quick summary of what the stateful_partition contains

**Reply by s0urce-c0de** [reply, marked as answer]
stateful_partition is a partition that stores lots of data. But i have an idea on how to enable developer mode. Note [here](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/main:src/platform2/runtime_probe/README.md) where it shows an alternative way to enter developer mode.
